include/pmsis/drivers/rtc.h

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) 2019 GreenWaves Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __PI_DRIVERS_RTC_H__
+#define __PI_DRIVERS_RTC_H__
+
+#include "pmsis/pmsis_types.h"
+#include <time.h>
+
+/**
+ * @ingroup groupDrivers
+ */
+
+/**
+ * @defgroup RTC RTC
+ */
+
+/**
+ * @addtogroup RTC
+ * @{
+ */
+
+/**
+ * \enum pi_rtc_mode_e
+ * \brief RTC working mode
+ */
+typedef enum
+{
+    PI_RTC_MODE_CALENDAR    = 0, /*!< RTC calendar mode. */
```

Comment:
I think this should be a bitfield as these features can all be activated at the same time in the RTC ?